recipes/mongo-cxx-driver/all/conanfile.py

```diff
@@ -0,0 +1,188 @@
+            self.output.warn("The recipe will force the cppstd to {}".format(self._minimal_std_version))
+            self._cmake.definitions["CMAKE_CXX_STANDARD"] = self._minimal_std_version
+        # FIXME: two CMake module/config files should be generated (mongoc-1.0-config.cmake and bson-1.0-config.cmake),
+        # but it can't be modeled right now.
+        # Fix should happen in mongo-c-driver recipe
+        if not os.path.exists("Findbson-1.0.cmake"):
+            self.output.info("Copying mongoc config file to bson")
+            shutil.copy("Findmongoc-1.0.cmake", "Findbson-1.0.cmake")
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+    def build(self):
+        self._patch_sources()
+
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="THIRD-PARTY-NOTICES", dst="licenses", src=self._source_subfolder)
+
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        if self.settings.os == "Windows":
+            for dll_file in glob.glob(os.path.join(self.package_folder, "bin", "*.dll")):
+                if os.path.basename(dll_file).startswith(("concrt", "msvcp", "vcruntime")):
+                    os.remove(dll_file)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        # FIXME: two CMake module/config files should be generated (mongocxx-config.cmake and bsoncxx-config.cmake),
+        # but it can't be modeled right now.
+        self.cpp_info.filenames["cmake_find_package"] = "mongocxx"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "mongocxx"
+        self.cpp_info.names["cmake_find_package"] = "mongo"
+        self.cpp_info.names["cmake_find_package_multi"] = "mongo"
+
+        # mongocxx
+        self.cpp_info.components["mongocxx"].names["cmake_find_package"] = "mongocxx_shared" if self.options.shared else "mongocxx_static"
+        self.cpp_info.components["mongocxx"].names["cmake_find_package_multi"] = "mongocxx_shared" if self.options.shared else "mongocxx_static"
+        self.cpp_info.components["mongocxx"].names["pkg_config"] = "libmongocxx"
+        self.cpp_info.components["mongocxx"].includedirs = [os.path.join("include", "lib", "v_noabi", "mongocxx")]
+        self.cpp_info.components["mongocxx"].libs = ["mongocxx" if self.options.shared else "mongocxx-static"]
+        if not self.options.shared:
+            self.cpp_info.components["mongocxx"].defines.append("MONGOCXX_STATIC")
+        if self.options.with_ssl:
+            self.cpp_info.components["mongocxx"].defines.append("MONGOCXX_ENABLE_SSL")
```

Comment:
```suggestion
```

Seems to be PRIVATE (only required at build time, on not exposed in public headers)